assets/css/screens/_html-view.scss

```diff
@@ -0,0 +1,9 @@
+body.html-view {
```

Comment:
I think this file should be `_keyword.scss` following the controller name and then we can do

```scss
body.keyword.html-view {
```

Later if there is more actions, we can also separate files like

```
keyword
  - _html-view.scss
  - _index.scss
```

But for now I think just `_keyword.scss` works for us.